modules/nf-core/diamond/blastp/meta.yml

```diff
@@ -42,6 +42,11 @@ input:
         used for in conjunction with the 'txt' out_ext option (--outfmt 6). See
         DIAMOND documnetation for more information.
 output:
+  - meta:
```

Comment:
Oops!

Reply:
Added some more info regarding this: Options:
        qseqid sseqid pident length mismatch gapopen qstart qend sstart send evalue bitscore